src/ansi_support.rs

```diff
@@ -5,76 +5,35 @@ use lazy_static::lazy_static;
 
 use crate::Result;
 
-/// Toggle virtual terminal processing.
+/// Enable virtual terminal processing.
 ///
-/// This method attempts to toggle virtual terminal processing for this
-/// console. If there was a problem toggling it, then an error returned.
-/// On success, the caller may assume that toggling it was successful.
+/// This method attempts to enable virtual terminal processing for this
+/// console. If there was a problem enabling it, then an error returned.
+/// On success, the caller may assume that enabling it was successful.
 ///
 /// When virtual terminal processing is enabled, characters emitted to the
 /// console are parsed for VT100 and similar control character sequences
 /// that control color and other similar operations.
-pub(crate) fn set_virtual_terminal_processing(yes: bool) -> Result<()> {
+fn enable_vt_processing() -> Result<()> {
     let mask = ENABLE_VIRTUAL_TERMINAL_PROCESSING;
 
     let console_mode = ConsoleMode::from(Handle::current_out_handle()?);
     let old_mode = console_mode.mode()?;
 
-    let new_mode = if yes {
-        old_mode | mask
-    } else {
-        old_mode & !mask
-    };
-
-    if old_mode != new_mode {
-        console_mode.set_mode(new_mode)?;
+    if old_mode & mask == 0 {
+        console_mode.set_mode(old_mode | mask)?;
     }
 
     Ok(())
 }
 
 lazy_static! {
     static ref SUPPORTS_ANSI_ESCAPE_CODES: bool = {
-        // Some terminals on windows like GitBash can't use WinaApi calls directly
```

Comment:
I would like to keep this comment.

Reply:
OK, it's been restored. I wasn't really thinking when I removed it :sweat_smile:.